lib/entice/logic/movement.ex

```diff
@@ -28,6 +29,9 @@ defmodule Entice.Logic.Movement do
         |> Map.put(Position, new_pos)
         |> Map.put(Movement, new_movement)
       end)
+    {:ok, %MapInstance{map: map, players: _}} = Entity.fetch_attribute(entity, MapInstance)    
```

Comment:
Does the player have a map-instance attribute? I thought it was for maps only...